src/main/kotlin/racingcar/Cars.kt

```diff
@@ -0,0 +1,20 @@
+package racingcar
+
+import racingcar.model.Car
+import racingcar.model.RaceCondition
+
+/**
+ * 자동차들의 객체 관리
+ * */
+class Cars(raceCondition: RaceCondition) {
+    private var _carList = (0 until raceCondition.carCount).map { Car() }.toList()
```

Comment:
생성자, 부생성자를 이용하는게 더깔끔해보여요.
```suggestion
class Cars(raceCondition: RaceCondition) {
    private var _carList = (0 until raceCondition.carCount).map { Car() }.toList()
```


아니면 팩토리 메소드를 만들어주는게 어떨까요?
```suggestion
class Cars private constructor(var _carList:List<Car>) 
.....
companion object{
        fun createCars(raceCondition: RaceCondition ):Cars{
            //TODO
        }
 }

```

Reply:
오 감사합니다!